.github/workflows/on_pull_request.yaml

```diff
@@ -15,7 +15,7 @@ on:
       - synchronize
 
 env:
-  FS_TAG: 0.0.0-pr${{ github.event.pull_request.number }}
+  FS_TAG: 0.0.0-pr${{ github.event.pull_request.number }}-${{ github.sha }}
```

Comment:
why add the sha here? The only problem is that the [delete tag script](https://github.com/zapier/kubechecks/blob/green-fix/.github/workflows/on_pull-request_closed.yaml#L30) stops working, and you can't use a wildcard there. could we fix the reason for this w/ a `imagePullPolicy: Always` somewhere?

Reply:
@djeebus, when I tested it, the pull policy `always` didn't work. When you make numerous changes to the PR, our cache pulls the old one, not the latest. I had to force this for EKS to pull the different tag. I'll revert it for now, but not sure there is any better solution.

Reply:
reverted it